markdown/org/docs/patterns/carlita/instructions/en.md

```diff
@@ -1,12 +1,63 @@
 
-## Maker's notes
+### Step 1: Draft and Cut the missing Lining and Facing Parts
+
+Some parts for Carlton require you to draft parts from the existing parts. The following are a list of what parts you need to draft, [see Carlton Cutting](/docs/patterns/carlton/cutting/) for details on how to cut them.
+
+#### Drafting
+##### Front facing and Front lining
+
+- Trace off of **Front** part.
+- Cut along the green line.
+- Add seam allowance (if including) along green line to both pieces
+- The piece with the lapel is now the **Front facing** part.
+- Trim the hem allowance (if included) of the remaining piece to seam allowance length
+- The remaining piece is now the **Front lining** part.
+
+<Note>
+
+The button and pocket markings are not needed for these parts so feel free not to trace them.
+
+</Note>
+
+<Warning>
+  
+Don't forget the seam allowance for these pieces when cutting if you are making these parts nett! 
```

Comment:
Just FYI I have never seen this work "nett" in english before and cannot find it anywhere.

Reply:
It might "net" without the extra "t" it means without seam allowances.